Doc/whatsnew/3.10.rst

```diff
@@ -1401,6 +1401,13 @@ subclasses with the :func:`runtime_checkable` decorator
 if they want runtime protocols.
 (Contributed by Yurii Karabas in :issue:`38908`)
 
+Importing from the ``typing.io`` and ``typing.re`` submodules will now emit
```

Comment:
It is worth to add it in the "Deprecated" section below.

Reply:
Initially I thought no because What's new in 3.8 should have it https://docs.python.org/3/whatsnew/3.8.html#deprecated, but surprisingly it doesn't. So yep we should add it as a proper reminder this time.